devsetup/edpm/edpm-play.yaml

```diff
@@ -218,3 +54,91 @@ spec:
+kind: OpenStackDataPlaneRole
+metadata:
+  name: deploy-edpm-role
+  namespace: openstack
+spec:
+  nodeTemplate:
+    networkConfig:
+      template: _EDPM_NETWORK_CONFIG_TEMPLATE_
+    managed: false
+    managementNetwork: ctlplane
+    ansibleUser: root
+    ansiblePort: 22
+    ansibleVars: |
+      registry_name: "quay.io"
+      registry_namespace: "tripleomastercentos9"
+      image_tag: "current-tripleo"
+      edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
+      edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
+      edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
+      edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+      gather_facts: false
+      enable_debug: false
+      # edpm firewall, change the allowed CIDR if needed
+      edpm_sshd_configure_firewall: true
+      edpm_sshd_allowed_ranges: _EDPM_SSHD_ALLOWED_RANGES_
+      # SELinux module
+      edpm_selinux_mode: enforcing
+      undercloud_hosts_entries: []
+      # edpm_hosts_entries role
+      extra_hosts_entries: []
+      vip_hosts_entries: []
+      hosts_entries: []
+      hosts_entry: []
+      plan: overcloud
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNode
+metadata:
+  name: deploy-edpm-node
+  namespace: openstack
+spec:
+  role: deploy-edpm-role
+  hostName: edpm-compute-0
+  ansibleHost: _COMPUTE_IP_
+  node:
+    networks:
+      - network: ctlplane
+        fixedIP: _COMPUTE_IP_
+    ansibleUser: root
+    openStackAnsibleEERunnerImage: _OPENSTACK_RUNNER_IMG_
+    ansibleVars: |
+      ansible_ssh_user: root
+      ctlplane_ip: 192.168.24.100
+      internal_api_ip: 192.168.24.2
+      tenant_ip: 192.168.24.2
+      fqdn_internal_api: '{{ ansible_fqdn }}'
+      service_net_map:
+        nova_api_network: internal_api
+        nova_libvirt_network: internal_api
+
+      # edpm_network_config
+      # Default nic config template for a EDPM compute node
+      # These vars are edpm_network_config role vars
+      edpm_network_config_hide_sensitive_logs: false
+      #
+      # These vars are for the network config templates themselves and are
+      # considered EDPM network defaults.
+      neutron_physical_bridge_name: br-ex
+      neutron_public_interface_name: eth0
+      ctlplane_subnet_cidr: 24
+      ctlplane_gateway_ip: 192.168.122.1
+
+      # edpm_nodes_validation
+      edpm_nodes_validation_validate_controllers_icmp: false
+      edpm_nodes_validation_validate_gateway_icmp: false
+
+      # 99-standalone-vars
+      ctlplane_dns_nameservers:
+      - 192.168.122.1
+      dns_search_domains: []
+      edpm_ovn_dbs:
+      - 192.168.24.1
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+  deployStrategy:
+    deploy: true
```

Comment:
can we just use the samples that I'm updating/adding in https://github.com/openstack-k8s-operators/dataplane-operator/pull/77 instead?

Reply:
yes, I'll update it